src/contrib/mlir/core/ngraph_dialect/fused_ops.td

```diff
@@ -32,14 +33,899 @@ def NGSqueezeOp :
+      "Attribute beginNormAxis, Attribute epsilon", [{
+        tblgen_state.addOperands({data, delta, scale});
+        tblgen_state.addAttribute("beginNormAxis", beginNormAxis);
+        tblgen_state.addAttribute("epsilon", epsilon);
+        tblgen_state.addTypes(res);
+      }]>,
+
+    OpBuilder<
+      "Builder *builder, OperationState &tblgen_state, ArrayRef<Type> res,"
+      "Value *data, Value *delta,"
+      "Attribute beginNormAxis, Attribute epsilon", [{
+        tblgen_state.addOperands({data, delta});
+        tblgen_state.addAttribute("beginNormAxis", beginNormAxis);
+        tblgen_state.addAttribute("epsilon", epsilon);
+        tblgen_state.addTypes(res);
+      }]>,
+  ];
+
+  let extraClassDeclaration = [{
+    void setBeginNormAxis(const Attribute& attr)  { this->setAttr("beginNormAxis", attr);}
+    void setEpsilonAxis(const Attribute& attr)    { this->setAttr("epsilon", attr);      }
+  }];
+}
+
+// HardSigmoid Op
+def NGHardSigmoid : 
+    NG_OneResult_Op<"hardSigmoid", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$data,
+               F32Attr:$alpha, F32Attr:$beta)>
+{
+    let summary = "Hard sigmoid op";
+    let description = [{
+      Parameterized, bounded sigmoid-like, piecewise linear
+      function. min(max(alpha*x + beta, 0), 1)
+    }];
+
+    let extraClassDeclaration = [{
+      void setAlpha(const Attribute& attr) { this->setAttr("alpha", attr); }
+      void setBeta(const Attribute& attr)  { this->setAttr("beta", attr); }
+    }];
+}
+
+// Gemm Op
+def NGGemmOp : 
+    NG_OneResult_Op<"gemm", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$A, NG_TensorType:$B, NG_TensorType:$C,
+              DefaultValuedAttr<F32Attr, "1.0">:$alpha,
+              DefaultValuedAttr<F32Attr, "1.0">:$beta,
+              DefaultValuedAttr<BoolAttr, "false">:$transA,
+              DefaultValuedAttr<BoolAttr, "false">:$transB)>
+{
+    let summary = "Gemm Op";
+    let description = [{
+      A' = transpose(A) if transA else A
+      B' = transpose(B) if transB else B
+      Compute Y = alpha * A' * B' + beta * C
+    }];
+    
+    let extraClassDeclaration = [{
+      void setAlpha(const Attribute& attr)   { this->setAttr("alpha", attr);}
+      void setBeta(const Attribute& attr)    { this->setAttr("beta", attr); }
+
+      void setTransA(const Attribute& attr)  { this->setAttr("transA", attr); }
+      void setTransB(const Attribute& attr)  { this->setAttr("transB", attr); }
+    }];
+}
+                  
+// GroupConv Op
+def NGGroupConvOp : 
+    NG_OneResult_Op<"groupConv", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$images, NG_TensorType:$filters, 
+               I64ArrayAttr:$strides,
+               I64ArrayAttr:$padBelow,
+               I64ArrayAttr:$padAbove,
+               I64Attr:$groups,
+               DefaultValuedAttr<PadTypeEnumAttr, 
```

Comment:
window_dilation_Strides and data_dilation_strides are missing. 
Does mlir not support dilation?